code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1074,6 +1074,11 @@
 	else if(can_be_firemanned(target))
 		fireman_carry(target)
 
+/mob/living/carbon/human/limb_attack_self()
```

Comment:
assume this will get signal'd like normal attack_self and call parent.

Reply:
Done